desc/protoparse/ast.go

```diff
@@ -729,17 +761,46 @@ type extensionRangeNode struct {
 type rangeNode struct {
 	basicCompositeNode
 	stNode, enNode node
-	st, en         int32
+	enMax          bool
```

Comment:
nit: `endMax`?

Reply:
Yeah, I'm not sure why I used `st` and `en` instead of `start` and `end`. I can do a quick rename for this and the two fields above.

Reply:
renamed